test/session-pool.js

```diff
@@ -16,48 +16,40 @@
 
 'use strict';
 
-var assert = require('assert');
-var common = require('@google-cloud/common');
-var delay = require('delay');
-var events = require('events');
-var extend = require('extend');
-var PQueue = require('p-queue');
-var proxyquire = require('proxyquire');
-var stackTrace = require('stack-trace');
-var through = require('through2');
-var timeSpan = require('time-span');
+const assert = require('assert');
+const common = require('@google-cloud/common');
+const events = require('events');
```

Comment:
this var is no longer used in the file and it can be removed

Reply:
Huh thought my lint fix run caught that, I'll run that again.